apps/judicial-system/web/src/routes/Court/Indictments/Advocates/SelectDefender.tsx

```diff
@@ -151,12 +155,53 @@ const SelectDefender: FC<Props> = ({ defendant }) => {
           />
         </Box>
         <InputAdvocate
+          advocateType="defender"
+          name={defendant.defenderName}
+          email={defendant.defenderEmail}
+          phoneNumber={defendant.defenderPhoneNumber}
+          onAdvocateChange={(
+            defenderName: string | null,
+            defenderNationalId: string | null,
+            defenderEmail: string | null,
+            defenderPhoneNumber: string | null,
+          ) =>
+            handleSetAndSendDefendantToServer({
+              defendantId: defendant.id,
+              defenderName,
+              defenderNationalId,
+              defenderEmail,
+              defenderPhoneNumber,
+            })
+          }
+          onAdvocateNotFound={setDefenderNotFound}
+          onEmailChange={(defenderEmail: string | null) =>
+            handleUpdateDefendantState({
```

Comment:
Nit: Do we have to update per property? Could we not have a single `onChange` or `onSave` that would handle both email and phone number change to reduce the number of props defined for `InputAdvocate`?

Reply:
We could and probably would if there were more than two properties, but since they are only two I decided to implement it this way. No correct way really.